.github/workflows/test.yml

```diff
@@ -5,19 +5,16 @@ on:
     branches:
       - master
   pull_request:
+    branches:
+      - master
 
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [10.x, 12.x, 14.x]
     steps:
       - uses: actions/checkout@v2
-      - name: Use Node.js ${{ matrix.node-version }}
+      - name: Use Node.js
         uses: actions/setup-node@v1
-        with:
-          node-version: ${{ matrix.node-version }}
```

Comment:
The used node version should be explicitly declared (12, which is used by google to run the function), else it will use the default declared be the action, which could change and break the tests.

Reply:
I had a look in the docs and the ubuntu 18 images have node 12 fixed.

But yes, defining the version will be the safe way to go. Will change this.